test/integration/targets/nxos_static_routes/tests/cli/merged.yml

```diff
@@ -0,0 +1,75 @@
+---
+    - debug:
+        msg: "Start nxos_static_routes merged integration tests connection={{ansible_connection}}"
+
+    - include_tasks: populate_config.yaml
+
+    - include_tasks: remove_config.yaml
+
+    - block:
+        - name: Merged
+          nxos_static_routes: &merged
+            config:
+              - vrf: trial_vrf
+                address_families:
+                  - afi: ipv4
+                    routes:
+                      - dest: 192.0.2.64/28
+                        next_hops:
+                          - forward_router_address: 192.0.2.22
+                            tag: 4
+                          
+                          - forward_router_address: 192.0.2.23
+                            route_name: merged_route
+                            admin_distance: 1
+
+              - address_families:
+                  - afi: ipv4
+                    routes:
+                      - dest: 192.0.2.16/28
+                        next_hops:
+                          - forward_router_address: 192.0.2.24
+                            route_name: new_route
+                            
+                      - dest: 192.0.2.80/28
+                        next_hops:
+                        - forward_router_address: 192.0.2.26
+                          tag: 12
+            state: merged
+          register: result
+
+        - assert:
```

Comment:
Can  you also add a test where you update an existing route?  What happens when you try and change a few parameters for a route that already exists?